packages/legend-query-builder/src/components/filter/QueryBuilderFilterPanel.tsx

```diff
@@ -898,6 +898,35 @@ export const QueryBuilderFilterPanel = observer(
       [handleDrop],
     );
 
+    const [{ addFilterDragOver }, filterTargetConnector] = useDrop<
```

Comment:
I believe `dropTargetConnector` is a special ref function, so you can do something like `dropTargetConnector(refForBlankNode)` - so no need to repeat code like this